### PR TITLE
The syndicate balloon now has a special suicide action

### PR DESCRIPTION
**Pull request by Contrabang**
## What Does This PR Do

The syndicate balloon now has a special suicide act, which makes you float into the air. The sound is the same one the ash drake uses when it launches. It drops everything except balloon, and the balloon is vaporized. Balloon thieves are now 100x more dangerous.

## Why It's Good For The Game
Adds more fun to the game

## Images of changes

(slightly outdated, clothes and stuff are left on the ground now). 

https://github.com/ParadiseSS13/Paradise/assets/91113370/3bb90189-98bc-49f6-9b15-9b744dc1179a

image

## Testing

See above, also tested with contractor balloon

## Changelog
:cl:
add: Syndicate balloons now have a special suicide action
/:cl:

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -123,6 +123,32 @@
...
 	lastused = world.time
 
+/obj/item/toy/syndicateballoon/suicide_act(mob/living/user)
+	. = ..()
+	if(!user)
+		return
+
+	user.visible_message("<span class='suicide'>[user] ties [src] around their neck and starts to float away! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+
+	// playsound(get_turf(user), 'sound/magic/tail_swing.ogg', 80, TRUE)
```
Comment (JimKil3) [inline code/game/objects/items/toys.dm:133]:
```suggestion
```
Don't think you're supposed to leave commented-out code in

**Review by JimKil3 — CHANGES_REQUESTED** [review]
Looks funny, I really hope I get to see this in-game at least once

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -123,6 +123,32 @@
 	user.visible_message("<span class='notice'>[user] plays with [src].</span>", "<span class='notice'>You [playverb].</span>")
 	lastused = world.time
 
+/obj/item/toy/syndicateballoon/suicide_act(mob/living/user)
+	. = ..()
+	if(!user)
```
[the remark this answers is not in the document]
Comment (Contrabang) [reply code/game/objects/items/toys.dm:128]: Always good to null check, all the other suicide_acts do it too

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (Contrabang) [reply code/game/objects/items/toys.dm:133]: true

**Comment by McRamon** [comment]
Lmao perfect

File: code/game/objects/items/toys.dm [hunk 3]
```diff
@@ -123,6 +123,34 @@
...
+		return
+
+	user.visible_message("<span class='suicide'>[user] ties [src] around [user.p_their()] neck and starts to float away! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+
+	playsound(get_turf(user),'sound/magic/fleshtostone.ogg', 80, TRUE)
```
Comment (lewcc) [inline code/game/objects/items/toys.dm:133]:
```suggestion
	playsound(get_turf(user), 'sound/magic/fleshtostone.ogg', 80, TRUE)
```

File: code/game/objects/items/toys.dm [hunk 4]
```diff
@@ -123,6 +123,34 @@
...
+	animate(holder_obj, pixel_z = 1000, time = 50)
+
+	for(var/obj/item/W in user)
+		user.unEquip(W)
+
+	user.notransform = TRUE
+	icon = null
+	invisibility = 101
+	QDEL_IN(user, 20)
+	QDEL_IN(src, 20)
```
Comment (lewcc) [inline code/game/objects/items/toys.dm:151]:
```suggestion
	QDEL_IN(user, 2 SECONDS)
	QDEL_IN(src, 2 SECONDS)
```

**Review by lewcc — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**